public/app/core/components/Login/LoginLayout.tsx

```diff
@@ -51,14 +51,11 @@ to{
 }`;
 
 export const getLoginStyles = (theme: GrafanaTheme2) => {
-  const bgColor = theme.isDark ? '#000' : theme.colors.background.canvas;
-
   return {
     container: css({
       minHeight: '100%',
       backgroundPosition: 'center',
       backgroundRepeat: 'no-repeat',
-      backgroundColor: bgColor,
```

Comment:
@torkelo I'm not really sure if there is a particular reason to have this style here, as it gets overwritten in Branding, but maybe I'm missing smth.

Reply:
Probably just a leftover